week3/assignment/week3_GayeonKim/src/App.jsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react'
+import './App.css'
+import Timer from './components/Timer/Timer'
+import Game from './components/Game/Game';
+import Ranking from './components/Ranking/Ranking';
+
+function App() {
+  //const [count, setCount] = useState(0)
+  const [view, setView] = useState('game'); // 일단 들어가자마자 보이는건 게임 state
+  const [isRunning, setIsRunning] = useState(false);
+  const [startTime, setStartTime] = useState(0);
+  const [playTime, setPlayTime] = useState(0);
+
+  const [level, setLevel] = useState(1); // 기본 레벨 1 (일단 심화과제 할 수 있으면 하기)
+
+
+  const handleViewChange = (newView) => {
+    setView(newView); // view 상태를 클릭한 버튼에 따라 변경
+    // console.log(`view state changed! ${view}`)
+  };
+
+  const startGame = () => {
+    setStartTime(new Date()); // 시작 한 시간 기록하기
+    setIsRunning(true); // 시작했다면 타이머를 굴려
+  }
+  
+//타이머 멈추는 함수 선언
+const stopGame = () => {
+  setIsRunning(false); //끝났으니 도는거 멈춰야됨
+  const gameData = {
+      startTime,
+      level,
+      playTime
+  };
+  localStorage.setItem(`game_${Date.now()}`, JSON.stringify(gameData));
```

Comment:
```suggestion
  const data = JSON.parse(localStorage.getItem('gameRecordsData')) || [];
  data.push(gameData);
  localStorage.setItem('gameRecords', JSON.stringify(data));
  ```
  이런식으로 수정해서 gameRecordsData라는 단일 키로 기록을 관리하면 코드가 더 깔끔해질 거예요 !